rust-toolchain.toml

```diff
@@ -1,4 +1,3 @@
-# Copyright Kani Contributors
 # SPDX-License-Identifier: Apache-2.0 OR MIT
```

Comment:
Can we also remove these?

Reply:
Sure, I thought we might want to keep it, but i guess its not something thats present in the upstream repo, so removing makes sense.